release-policies.md

```diff
@@ -47,6 +47,8 @@ Patches are published at the [.NET Website](https://dotnet.microsoft.com/downloa
 
 As the end of support nears for a given .NET version, we strongly recommend you move to a newer, supported version of .NET. .NET releases that have reached end of support do not get security patches. Continuing to use an unsupported version will expose you to security vulnerabilities.
 
+End of support also applies to source that targets unsupported .NET versions. You must target a supported .NET version (via the `TargetFramework` property), for both apps and libraries. The .NET SDK produces warnings when you target out of support versions to help identify these cases. The SDK does not prevent you from targeting unsupported versions, however, your configuration will be considered unsupported.
```

Comment:
>  You must target a supported .NET version (via the `TargetFramework` property), for both apps and libraries

This is incorrect. We specifically support targeting old TF in libraries.

Reply:
> You must target a supported .NET version (via the `TargetFramework` property), for both apps and libraries. 

Just to add to that, we have specifically given 1P partner guidance to no waste their time upgrading .NET Framework/.NET Core version in library projects and told them that upgrading application/services/unit tests is what they should focus on.

Also, it just doesn't make sense to say "all versions of .NET Standard are supported to target in perpetuity" and simultaneously say that targeting the frameworks that implement them from the same projects is unsupported.